src/components/footer.jsx

```diff
@@ -18,8 +18,8 @@ function Footer() {
 
   const primaryLinks = links.primary;
   const primaryLinksLength = primaryLinks.length;
-  const primaryLinksLabels = t('navigation.dropdowns').map((element) => element.linkLabels).flat().concat(t('navigation.parentLinkLabels'));
-  const primaryLinksLabelsLength = primaryLinksLabels.length;
+  const primaryLinksLabels = t('navigation.dropdowns').map && t('navigation.dropdowns').map((element) => element.linkLabels).flat().concat(t('navigation.parentLinkLabels'));
```

Comment:
The logic in this file has changed through my PRs. I would recommend merging dev into this branch and making sure what you have here works.